### ISSUE TITLE
Missing Accessibility Elements for Web Content Accessibility Guidelines

### ISSUE THREAD

**Issue by Pic003Developer** [PrintStudentDeveloper/DarkSkiesWebsite#1]
A new version has been upload containing the required Updates
